src/loaders/jobs/taskfarmer/task_generator.py

```diff
@@ -49,10 +49,11 @@
 256 cores.
 '''
 
-TOOLS_AVAILABLE = ['gtdb_tk', 'checkm2', 'microtrait', 'mash']
+TOOLS_AVAILABLE = ['gtdb_tk', 'checkm2', 'microtrait', 'mash', 'eggnog']
```

Comment:
Is this going to confuse / interact with the rest of the pipeline?

Reply:
no, I don't think so. The parser has its own defined tools. Even if we run eggnog on legit collections, the parser will skip this tool.

Reply:
and running the tasks won't be affected?

Reply:
No. Parser will just skip the result of this tool. And plus, we are running this tool on a custom collection.

Reply:
:+1: